.azure/modules/vnet/main.bicep

```diff
@@ -4,6 +4,9 @@ param namePrefix string
 @description('The location where the resources will be deployed')
 param location string
 
+@description('Tags to apply to resources')
```

Comment:
Apply this description to all tag params? 
Or remove 🤷🏼 ? 

Dunno how consistent we have been with descriptions, I have been slack at least 🙈

Reply:
yeah, we should apply @decsription it to all tbh 🤔 Can do that in the same PR. What you think?

Reply:
Yeah, sure 🙌🏼

Reply:
Added all the things